Library/Homebrew/cmd/test-bot.rb

```diff
@@ -789,6 +789,11 @@ def test_bot
         version = formula.pkg_version
         bintray_package = Bintray.package formula_name
 
+        if system "curl", "-I", "--silent", "--fail", "--output", "/dev/null",
+                  "#{BottleSpecification::DEFAULT_DOMAIN}/#{bintray_repo}/#{filename}"
+          raise "#{filename} is already published. Please remove it manually."
```

Comment:
Would be good to provide the full URL to the file so people can click through there to remove it.